### PR TITLE
Adjusts Asimov to make it more interesting.

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request

Adjusts Asimov to make it more interesting.

## Why It's Good For The Game

we've been using bog standard asimov for 20 years let's spice it up a little

## Changelog

:cl:
qol: Adjusts Asimov to make it more interesting.
/:cl:

### CODE REVIEW

**Comment by Iamgoofball** [comment]
ok lemme fix that

**Comment by GuillaumePrata** [comment]
I like the change but it would be nice to rename/reword it to to an Asimov variant, like Nutmov does, and then retire the default Asimov.

Asimov has been a staple of SS13 for years, it cross through different codebases and server's policy but the laws were always on the same order.
Having people come from other servers/from a break on SS13 and get confused between both the old and new version will just get in the way of a good change.

**Comment by Mothblocks** [comment]
This is the absolute lowest effort "why this is good for the game" in a while

This is an enormous change that is heavily influenced by policy and there needs to be a hell of a lot more discussion going on than this, especially with zero attempt to explain why this is beneficial

**Comment by dopamiin0** [comment]
image

**Comment by Iamgoofball** [comment]
chrome_F0iaXSSQD5
where's the problem here? this is a worker standing up for his rights

**Comment by optimumtact** [comment]
Also, policy issue laws are configurable
